nifti_laser_filtering:
    scan_filter_chain:

    -   name: NearFilter
        type: nifti_laser_filtering/NearFilter

    -   name: TradrLaserShadowFilter
        type: nifti_laser_filtering/TradrLaserShadowFilter
        params:
            min_angle_close: 0.10  # radians
            min_angle_far: 0.10  # radians
            far_distance_threshold: 0.7  # meters

    -   name: SelfEdgeLaserFilter
        type: nifti_laser_filtering/SelfEdgeLaserFilter

